app/bundles/EmailBundle/Translations/en_US/messages.ini

```diff
@@ -172,6 +172,10 @@ mautic.email.config.mailer.rabbitmq_timeoutconnect="Connection timeout"
 mautic.email.config.mailer.rabbitmq_timeoutpush="Push timeout"
 mautic.email.config.mailer.rabbitmq_timeoutpull="Pull timeout"
 mautic.email.config.mailer.rabbitmq_heartbeatpull="Pull Heartbeat"
+mautic.email.config.mailer.rabbitmq_use_ssl="Use SSL"
```

Comment:
nitpick: Could you make these descriptions say "TLS/SSL" instead of just SSL? Nobody is actually using SSL anymore.